molgenis/molgenis-emx2
# feat(directory): add separate ontologies schema

how to test:
- Go to https://preview-emx2-pr-3560.dev.molgenis.org and see the new Ontologies schema. Check also directory-demo, schema editor => check fields of ontology (array) data type have now a references to the Ontologies schema.
NB! With this change the filters are broken. 

todo:
- [x] When adding `database.dropSchemaIfExists(ONTOLOGIES); ` to TestLoaders.java the test fails with `Transaction failed: Drop schema failed: Drop table failed: cannot drop column name of table "Ontologies"."ImagingModalities" because other objects depend on it. Details: trigger DEL_testOntologies_Collections_imaging_modality on table "Ontologies"."ImagingModalities" depends on column name of table "Ontologies"."ImagingModalities"`

I don't know how to fix this....

=> It turned out that there was another schema using the ontology schema...

- [x] fix Directory App => ontology data should be retrieved from Ontologies schema
